foxpuppet/foxpuppet.py

```diff
@@ -1,23 +1,26 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+"""FoxPuppet object for browser interaction."""
 
 from __future__ import absolute_import
 
 from foxpuppet.windows import WindowManager
 
 
 class FoxPuppet(object):
-    """
-        Class that sets up the interface for interacting with the Firefox
-        browser.
+    """Set up the interface for interacting with the Firefox browser.
 
-        :param selenium: WebDriver object
-        :type selenium:
-            :py:class:`~selenium.webdriver.remote.webdriver.WebDriver`
+    Args:
+        selenium: (:obj:`WebDriver`): Firefox WebDriver object.
```

Comment:
This drops the link to the Selenium docs. Could you restore this by using:

```
selenium: (:py:class:`~selenium.webdriver.remote.webdriver.WebDriver`): Firefox WebDriver object.
```

This comment should be applied to all arg and return types.